### DISCUSSION TITLE
Use mining pool farming on many machines

### DISCUSSION THREAD

**Discussion by qq597353** [Chia-Network/chia-blockchain#8105] [Q&A]
After reading this article, it didn't mention how to add multiple harvesters to the mining pool for farming
[Farming on Many Machines](https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines)

```
                                                   _____  Harvester 1 (certificate A)
                                                 /
other network peers  --------   Main machine (CA) ------  Harvester 2 (certificate B)
                                                 \_____  Harvester 3 (certificate C)
```

What kind of structure should it be?

How to join the mining pool?

Whether “Main machine (CA)” joins the mining pool or “ Harvester x” joins the mining pool?

Who can help me?😭

**Comment by 21497724** [comment]
Farmer run on main machine and join to pool, harvesters running on remote machine and connect to farmer.

**Reply by qq597353** [reply]
Can you elaborate on the command to join the mining pool on the farmer's host?

**Reply by rumblersack** [reply]
I'm having trouble understanding harvesting on many machines as well. I am doing the whole multiple harvesters connected to main machine but I am seeing a bit of a problem. My total plot count shows 247 plots however at the top of the Plots tab it shows my Total Plots Size is only 8.216TiB, I would think this should say 24.7TiB. Also, when plugging my launcher ID into Space Pool it shows an estimate of plots to be 8TiB. My main machine has only produced 82 plots so I understand where the 8.216 TiB is coming from, but the pool doesn't seem to be accounting for all 247 plots.

**Reply by 21497724** [reply]
before 1.2.0, the main machine only show local plots, after 1.2.0 can show remote havester's plots at main machine.

**Comment by BrandtH22** [comment]
@qq597353 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
